### PR TITLE
feat(search): show individual match results when available

### PR DESCRIPTION
**Pull request by davidlougheed**

### CODE REVIEW

**Review by gsfk — COMMENTED** [review]
Works as expected. I'm assuming more stuff will be filled in later. 

I had to click "Sign in" on the portal page that popped up, even though I was already signed in, although I wasn't forced to login a second time. But I assume this too won't be an issue later.

**Comment by davidlougheed** [comment]
yeah the idea is to have something relatively straightforward to show that we're moving towards integration of auth and gradual permissions, and to slowly merge it with beacon/the explore page as we go.

File: src/styles.css
Comment (v-rocheleau) [inline src/styles.css:1]: It was not clear to me at first that the `Individuals` element was clickable, I think a hover class would help with that.

**Review by v-rocheleau — CHANGES_REQUESTED** [review]

File: src/js/components/Search/SearchResults.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ const SearchResults = () => {
   const experimentCount = useAppSelector((state) => state.query.experimentCount);
   const experimentChartData = useAppSelector((state) => state.query.experimentChartData);
   const individualCount = useAppSelector((state) => state.query.individualCount);
+  const individualMatches = useAppSelector((state) => state.query.individualMatches);
```
[the remark this answers is not in the document]
Comment (davidlougheed) [reply src/js/components/Search/SearchResults.tsx:11]: i agree

File: src/styles.css
Comment (davidlougheed) [reply src/styles.css:1]: sounds good!

**Review by v-rocheleau — APPROVED** [review]
LGTM!

**Pull request merged**